services/testing/nightwatch/drupal.go

```diff
@@ -44,3 +50,49 @@ func checkDrupalVersion() error {
 
 	return errors.New("The Drupal version (" + drupalVersion + ") is not supported for Nightwatch tests. Drupal version must be " + validDrupalRange + ". See: https://github.com/drupal/drupal/blob/8.6.x/core/tests/README.md#nightwatch-tests for more information.")
 }
+
+// checkNightwatchConfig scans nightwatch config files in Drupal Core to see
+// if they contain settings that we need. If not, we provide info for the user
+// to manually fix those, until such time as we have Drupal patches ready
+func checkNightwatchConfig() error {
+	ok := true // If false at the end of checking, the program will exit with an error
+	pr := conf.GetProjectPath()
+	dr := conf.GetConfig().Drupal.Path
+
+	nightwatchConfPath := filepath.Join(pr, dr, "core/tests/Drupal/Nightwatch/nightwatch.conf.js")
+	nightwatchOk := fs.Contains(nightwatchConfPath, "acceptInsecureCerts: true")
+	utils.DebugString("nightwatchConfPath: " + nightwatchConfPath)
+	utils.DebugString("nightwatchOk: " + strconv.FormatBool(nightwatchOk))
+
+	if !nightwatchOk {
+		ok = false
+		fmt.Println(Red("Tokaido tests via HTTPS, but Druapl's Nightwatch config only supports HTTP."))
+		fmt.Println(Red("To fix this, you need to add 'acceptInsecureCerts: true' to the desiredCapabilities "))
+		fmt.Println(Red("section of "), nightwatchConfPath)
+		fmt.Println()
+		fmt.Println(Yellow("Please see docs.tokaido.io for more information"))
```

Comment:
Once we have a specific page set up, we should link it here, or provide enough detail to fix without switching context

Reply:
Yup - with the new docs site I'll be going through all these old doc links and making sure they're accurate before we release 1.9